cpu/kw2x/include/cpu_conf.h

```diff
@@ -72,6 +72,75 @@ extern "C"
 #define LPTIMER_CLKDIS()                 (SIM->SCGC5 &= ~SIM_SCGC5_PTMR_MASK) /**< Disable LPTMR0 clock gate */
 #define LPTIMER_CNR_NEEDS_LATCHING       1 /**< LPTMR.CNR register do not need latching */
 
+#define KINETIS_PMCTRL SMC->PMCTRL
+#define KINETIS_PMCTRL_SET_MODE(x)      (KINETIS_PMCTRL = SMC_PMCTRL_STOPM(x) | SMC_PMCTRL_LPWUI_MASK)
+#define KINETIS_PMPROT_UNLOCK()         (SMC->PMPROT |= SMC_PMPROT_ALLS_MASK | SMC_PMPROT_AVLP_MASK)
+
+/**
+ * @name STOP mode bitfield values
+ * @{
+ */
+/** @brief Normal STOP */
+#define KINETIS_POWER_MODE_NORMAL       (0b000)
+/** @brief VLPS STOP */
+#define KINETIS_POWER_MODE_VLPS         (0b010)
+/** @brief LLS STOP */
+#define KINETIS_POWER_MODE_LLS          (0b011)
+/** @} */
+
+/**
+ * @brief IRQn name to enable LLWU IRQ in NVIC
+ */
+#define KINETIS_LLWU_IRQ                LLWU_IRQn
+
+/**
+ * @brief LLWU module is always on.
+ */
+#define LLWU_UNLOCK()
+
+/**
+ * @brief Internal modules whose interrupts are mapped to LLWU wake up sources.
+ *
+ * Other modules CAN NOT be used to wake the CPU from LLS or VLLSx power modes.
+ */
+typedef enum llwu_wakeup_module {
```

Comment:
I think it is at all Kinetis M4 equal, some are reserved but the order is the same. Can we take it as a default?